tests/integration/raftlog.py

```diff
@@ -178,10 +178,10 @@ def reset(self):
     def read(self):
         while True:
             try:
-                entry = RawEntry.from_file(self.logfile)
+                ety = RawEntry.from_file(self.logfile)
```

Comment:
what was wrong with `entry` ?

Reply:
Looks like main function, `__main__`, does not define variable scope in Python. All variables in this procedure are global variables. So, linter was complaining about shadowing variables from the outer scope. What a horrible language. 

I added another commit to move `__main__` into another function, so, variables will be part of function scope. Probably, this is clear for everybody and less likely to fall in a trap in future.